flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/GeneratedAggregations.scala

```diff
@@ -67,7 +67,7 @@ abstract class GeneratedAggregations extends Function {
     *                     aggregated results
     * @param input        input values bundled in a row
```

Comment:
Please add `@returns` documentation

Reply:
Please change `retract()` accordingly to keep the interfaces consistent.

Alternatively, you can also revert the changes on `GeneratedAggregations` which is only an internal interface. The Table API `org.apache.flink.table.functions.AggregateFunction` does not support immutable accumulators, so this change has no effect until the public interface is changed. If you revert, we only need to touch `AggregateAggFunction` and not the code generator.